### PR TITLE
New package: Lidarr-0.5.0.583

### PR DESCRIPTION
**Pull request by youngjinpark20**
https://github.com/void-linux/void-packages/issues/6685

Radarr and Sonarr are similar, so I'll get around to doing that soon :)

### CODE REVIEW

File: srcpkgs/Lidarr/template [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+license="GPL-3.0-or-later"
+homepage="http://lidarr.audio/"
+distfiles="https://github.com/lidarr/Lidarr/archive/v${version}.tar.gz"
+checksum="58b814975b568fccaf3f68bfc37b9c7d81bea123a5c5f4e57fa1fd08b16bf450"
+system_accounts="lidarr"
+lidarr_homedir="/var/lib/lidarr"
+
+post_extract() {
+	cd src/ExternalModules
+	git clone https://github.com/Sonarr/CurlSharp
```
Comment (maxice8) [inline srcpkgs/Lidarr/template]: add it to distfiles

Comment (youngjinpark20) [reply srcpkgs/Lidarr/template]: They need to latest version within their repo that's not a relesae version. I guess I should request it?

**Comment by youngjinpark20** [comment]
updated with distfile

File: srcpkgs/Lidarr/template [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Template file for 'Lidarr'
+pkgname=Lidarr
+version=0.5.0.583
+revision=1
+make_dirs="
```
Comment (maxice8) [inline srcpkgs/Lidarr/template]: make_dirs= go below the variables block

Comment (youngjinpark20) [reply srcpkgs/Lidarr/template]: I wasn't sure exactly what you meant. Is this it?

Comment (maxice8) [reply srcpkgs/Lidarr/template]: yes.

**Comment by maxice8** [comment]
Please mark it nocross with  `System.DllNotFoundException: libgdiplus.so.0` and broken on i686

**Comment by youngjinpark20** [comment]
I think i did it properly, but definitely might not have

**Pull request closed without merging**